app/bundles/CoreBundle/Helper/TemplatingHelper.php

```diff
@@ -39,21 +40,24 @@ public function __construct(Kernel $kernel)
     /**
      * Retrieve the templating service.
      *
-     * @return \Symfony\Bundle\FrameworkBundle\Templating\DelegatingEngine
+     * @return DelegatingEngine
+     *
+     * @throws \Exception
      */
     public function getTemplating()
     {
         if (defined('IN_MAUTIC_CONSOLE')) {
-            //enter the request scope in order to be use the templating.helper.assets service
-            $this->container->enterScope('request');
-            $this->container->set('request', new Request(), 'request');
+            // Use the request scope in order to be the templating.helper.assets service used
+            $this->container->set('request', new Request());
         }
 
         return $this->container->get('templating');
     }
 
     /**
      * @return TemplateNameParser
+     *
+     * @throws \Exception
```

Comment:
Why was this added? I can't find where it throws an exception.

Reply:
It is autodetected by code inspection in PhpStrom. I tried to search for this too  and not found. Removed